package.json

```
@@ -33,7 +33,7 @@
     "typescript": "~4.7.4"
   },
   "engines": {
-    "node": ">=14.15.0"
+    "node": ">=16.1"
```

Let's go for the proper semver targeting the current LTS versions (ignoring 14):
```suggestion
    "node": "^16.13 || ^18.12"
```

References:
- [Node v16.13.0 (LTS)](https://nodejs.org/en/blog/release/v16.13.0)
- [Node v18.12.0 (LTS)](https://nodejs.org/en/blog/release/v18.12.0)